js/okex.js

```diff
@@ -3203,7 +3203,7 @@ module.exports = class okex extends Exchange {
                 'amount': this.safeNumber (entry, 'sz'),
             });
         }
-        return result;
+        return this.filterBySymbolSinceLimit (result);
```

Comment:
We have to pass result, symbol, since and limit to `filterBySymbolSinceLimit`, otherwise it is not filtering anything.
Suppose what happens, if the user supplies any combination of `symbol` + `since` + `limit`. In all cases, we have to apply the filtering according to the user's request. If the user supplied `BTC-USDT-SWAP` under the symbol, we have to return just those histories, not all of them. The same is true for since and limit. If the user asked 100 entries, we should not return more than that. If the user asked for all entries after a certain timestamp – we should not return those results that were before the requested time.

Reply:
This is outlined here: https://github.com/ccxt/ccxt/wiki/Manual#pagination

The rule of thumb is – all methods returning an array of results must apply some kind of pagination.

Reply:
Find all occurences of `filterBySymbolSinceLimit` across the lib, and try copying the usage from there.

Reply:
P.S. we will have to do the same in all similar methods where filtering was left out.